### PR TITLE
docs(cli): normalize help messages; pretty cypher-like headers

### PR DESCRIPTION
**Pull request by akollegger**
### Please check if the PR fulfills these requirements
- [√] The commit message follows our [guidelines](https://github.com/neo4j-devtools/relate/blob/master/CONTRIBUTING.md)
- [-] Tests for the changes have been added (for bug fixes / features)
- [√] Docs have been added / updated (for bug fixes / features)

### What kind of change does this PR introduce?

Improved help messages for @relate/cli.

### What is the current behavior?

Good initial help.

### What is the new behavior?

Improved help, with some prettier headers.

### Does this PR introduce a breaking change?

This does not change any behaviors.

### CODE REVIEW

**Review by nglgzz — COMMENTED** [review]
This looks so much better than before, thank you! 

Just a couple of comments/questions.

File: packages/cli/README.md [hunk 1]
```diff
@@ -51,14 +51,14 @@ relate$ relate --help
...
 
-* [`relate app`](./docs/app.md) - Manage Graph Apps
+* [`relate app`](./docs/app.md) - Single page web apps.
 * [`relate autocomplete`](./docs/autocomplete.md) - display autocomplete installation instructions
-* [`relate backup`](./docs/backup.md) - Create a new backup
-* [`relate db`](./docs/db.md) - Manage Neo4j Databases
-* [`relate dbms`](./docs/dbms.md) - Manage Neo4j DBMSs
-* [`relate environment`](./docs/environment.md) - Manage relate environments
-* [`relate extension`](./docs/extension.md) - Manage relate extensions
+* [`relate backup`](./docs/backup.md) - Archives of Relate-managed resources (though not Neo4j DBMS; see "db:dump").
```
[the remark this answers is not in the document]
Comment (akollegger) [reply packages/cli/README.md:56]: Yes, good catch!

File: packages/cli/docs/environment.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ Manage relate environments
 
 ## `relate environment:api-token [CLIENTID]`
 
-Generate API token for a relate web API calls
+Generate API token for GraphQL access to Relate
```
[the remark this answers is not in the document]
Comment (akollegger) [reply packages/cli/docs/environment.md:15]: Good clarification. Thanks!

[the remark this answers is not in the document]
Comment (akollegger) [reply packages/cli/docs/environment.md:15]: Perhaps "Generate API token for HTTP access to Relate"?

File: packages/cli/package.json [hunk 3]
```diff
@@ -32,17 +32,21 @@
         "@oclif/plugin-help": "3.2.0",
         "@relate/common": "^1.0.2-alpha.11",
         "@relate/types": "^1.0.2-alpha.9",
+        "chalk": "4.1.0",
         "cli-ux": "5.5.0",
         "cross-env": "7.0.2",
         "enquirer": "2.3.6",
         "fs-extra": "9.0.1",
         "graphql": "15.1.0",
+        "indent-string": "4.0.0",
```
[the remark this answers is not in the document]
Comment (akollegger) [reply packages/cli/package.json:41]: I had extracted code from oclif which needed them, but then dropped that implementation path. They can be removed.

File: packages/cli/docs/environment.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (huboneo) [reply packages/cli/docs/environment.md:15]: Indeed, anything that is not a request to `/health` or `/static` needs the token if enabled https://github.com/neo4j-devtools/relate/blob/master/packages/web/src/auth/services/auth.service.ts#L37

**Review by nglgzz — APPROVED** [review]
Looks good to me

**Pull request merged**